dotnetv3/Lambda/Scenarios/LambdaScenarioCommon/UIWrapper.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[Lambda.dotnetv3.LambdaScenarioCommon.UIWrapper]
+namespace LambdaScenarioCommon;
+public class UIWrapper
+{
+    public readonly string SepBar = new('-', Console.WindowWidth);
+
+    /// <summary>
+    /// Show information about the Amazon S3 Basics scenario.
+    /// </summary>
+    public void DisplayLambdaBasicsOverview()
+    {
+        Console.Clear();
+
+        DisplayTitle("Welcome to AWS Lambda Basics");
+        Console.WriteLine("This example application does the following:");
+        Console.WriteLine("\t1. Create an Amazon Identity Management role that will be assumed by the functions we create.");
+        Console.WriteLine("\t2. Attach an IAM role policy that has Lambda permissions.");
+        Console.WriteLine("\t3. Create a Lambda function that increments the value passed to it.");
+        Console.WriteLine("\t4. Calls the increment function and passed a value.");
+        Console.WriteLine("\t5. Updates the code so that the function is a simple calculator.");
+        Console.WriteLine("\t6. Calls the calculator function with the values entered.");
+        Console.WriteLine("\t7. Deletes the Lambda function.");
+        Console.WriteLine("\t7. Detaches the IAM role policy.");
+        Console.WriteLine("\t8. Deletes the IAM role.");
+        PressEnter();
+    }
+
+    /// <summary>
+    /// Show information about the Amazon S3 Basics scenario.
+    /// </summary>
+    public void DisplayTransferUtilityBasicsOverview()
+    {
+        Console.Clear();
+
+        DisplayTitle("Amazon S3 Transfer Utility Basics");
+        Console.WriteLine("This program shows how to use the Amazon S3 Transfer Utility.");
+        Console.WriteLine("It performs the following actions:");
+        Console.WriteLine("\t1. Upload a single object to an S3 bucket.");
+        Console.WriteLine("\t2. Upload an entire directory from the local computer to an\n\t  S3 bucket.");
```

Comment:
```suggestion
        Console.WriteLine("\t2. Upload an entire directory from the local computer to an  S3 bucket.");
```

Reply:
These aren't random letters. They represent a new line and a tab. They belong there.